tests/common/fixtures/advanced_reboot.py

```diff
@@ -39,6 +39,16 @@ def __init__(self, request, duthost, ptfhost, localhost, tbinfo, creds, **kwargs
             "Please set rebootType var."
         )
 
+        if duthost.facts['asic_type'] == 'vs':
```

Comment:
>duthost.facts['asic_type'] == 'vs': [](start = 11, length = 35)

Better to check `x86_64-kvm_x86_64-r0` in platform

Reply:
Checking `x86_64-kvm_x86_64-r0` in platform instead.